src/lookup-table/services/LookupTable.service.ts

```diff
@@ -89,6 +145,25 @@ const LookupTableService = (structure: LookupTableServiceProps) => {
     fieldMaps = [...dbFieldMap, ...fieldMaps].filter(
       ((set) => (obj: any) => (set.has(obj.label) ? false : set.add(obj.label)))(new Set())
     )
+
+    return {
+      onlyInDb: dbFieldMap.filter(_comparer(fieldMaps)),
+      onlyInCsv: fieldMaps.filter(_comparer(dbFieldMap)),
+      inBoth: dbFieldMap.filter((current: any) =>
+        fieldMaps.filter((csvMap: any) => csvMap.label === current.label)
```

Comment:
I am not sure if it's useful to show user what's been changed without confirming with them first. This seems like a new spec, would be good if you can message me if there is a change like that first, so we can brainstorm how it works.

What should happen, imo, is before we do the update, we would have another end point that checks the diff and tells the user what will be changed (but that's out of scope for now))

Reply:
I am keeping the method since as of your description we might have a use for this later.